cli/mender-artifact/main.go

```diff
@@ -59,6 +59,38 @@ func main() {
 	}
 }
 
+// Copied from urfave/cli/template.go
```

Comment:
Maybe we could have updated [our fork](https://github.com/mendersoftware/cli/commit/2eb686fe967b14b3e204f0eff66427e1fdafcf6d) instead?

Reply:
Hmm, possibly, I'm not sure If I think it's a better solution though. Let's try and keep the forks as similar as possible

Reply:
Agree, this is a custom template, we should not modify the original I think.